### PR TITLE
[Bugfix:InstructorUI] Fix Filter Activity by Poll Count

### PR DESCRIPTION
**Pull request by NiaHeermance**
### What is the current behavior?
Filtering by poll count highlights all rows as green.

### What is the new behavior?
Now only rows where the poll count is greater than or equal the specified value are highlighted as green.

### Images
image

### CODE REVIEW

File: site/ts/activity-dashboard.js [hunk 1]
```diff
@@ -130,10 +130,13 @@ export function applySettings() {
         const s_grad_sub = data[i].gradeable_submission;
         const s_forum_view = data[i].forum_view;
         const s_forum_post = data[i].forum_post;
-        const s_num_polls = data[i].num_poll_responses;
+        let s_num_polls = data[i].num_poll_responses;
         const s_off_hours = data[i].office_hours_queue;
-        let flag = false;
+        if (s_num_polls === null) {
+            s_num_polls = 0;
+        }
```
Comment (williamjallen) [inline site/ts/activity-dashboard.js:137]: You could use the [null(ish)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) coalescing operator for this.

**Review by Nicc041 — APPROVED** [review]
Other than what William said, the code works as expected; the rows are filtered by the numbers of polls answered and turn red or green.

**Review by ANS1514 — APPROVED** [review]

**Comment by bmcutler** [comment]
@NiaHeermance 
please update this PR with William's feedback

File: site/ts/activity-dashboard.js (hunk 1 above)
Comment (NiaHeermance) [reply site/ts/activity-dashboard.js]: Unfortunately, as we figured out in the meeting yesterday, our linter doesn't recognize that operator. We need to update the linter.

**Pull request merged**